lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/failed.yaml

```diff
@@ -85,7 +85,7 @@ states:
                   successful: false
         checkJourneyContext:
           rfc:
-            targetState: CRI_TICF_BEFORE_COULD_NOT_CONFIRM_DETAILS
+            targetState: CRI_TICF_BEFORE_COULD_NOT_UPDATE_DETAILS_RFC
```

Comment:
Do we also need to check the feature flag `updateDetailsAccountDeletion` ?

Reply:
not sure as we don't have any default page with delete option.

Reply:
I think we do - otherwise we might reach this without it being enabled?